Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js

```diff
@@ -102,6 +102,7 @@ localizedStrings["0 Console warnings"] = "0 Console warnings";
 localizedStrings["1 match"] = "1 match";
 localizedStrings["1080p"] = "1080p";
 localizedStrings["2D"] = "2D";
+localizedStrings["2D (Offscreen)"] = "2D (Offscreen)";
```

Comment:
Confused as to why this change was made?

Reply:
The strings file is generated using a script, so this indicates that these strings were missing from the file previously.